app/helpers/application_helper.rb

```diff
@@ -1,4 +1,14 @@
 module ApplicationHelper
+
+  # Used to generate links to the policies finder
+  def policies_filter_path(query_params = nil)
```

Comment:
I am not convinced by this method, or at least its name. It is no longer the policies filter page, it is internally known as the policies finder, so we should call it as such. It also seems overkill to abstract it out into a method when it is only used once, especially that you have added a code path which isn't ever called.